satellite/audit/reservoir.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+	"storj.io/storj/pkg/pb"
+)
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Segments []*pb.RemoteSegment
+	IsVetted bool
+	NumSlots int
+}
+
+type reservoirConfig struct {
+	numVettedSlots   int
+	numUnvettedSlots int
```

Comment:
`slotsForVetted` and `slotsForUnvetted` sound clearer to me.

Reply:
Otherwise, it is easy to read as "number of (un)vetted slots", which would be wrong.

Reply:
I can change it, but also do I need to keep this? it didn't seen like we wanted to distinguish between slots for vetted and unvetted nodes after all in today's architecture meeting

Reply:
Yeah, I think we can simplify it to a single "slots" config.